ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -73,6 +79,38 @@ export default class StatusDropdown extends React.Component {
     };
   };
 
+  getNumberOfWeekendDays(createdAt, dueDate) {
+    const createdDate = new Date(createdAt.getTime());
+    const friday = 5;
+    const saturday = 6;
+    const sunday = 0;
+
+    // this sets the hour for both day created and due date to 00:00
+    // in order to have a full day availabe with the status "Today" to work on the alert
+
+    createdDate.setHours(0, 0, 0, 0);
+    dueDate.setHours(0, 0, 0, 0);
+
+    if (createdDate.getDay() === sunday) {
```

Comment:
It could be good to move `createdDate.getDay()` in a constant since it's used 3 times in this function.